README.md

```diff
@@ -1 +1,40 @@
-# Hello-Microverse-Reviewable
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello World
+
+> A basic Hello Microverse Project with HTML and CSS. It has Linters Implemented and ignored redundant files using .gitignore
+
+![screenshot]()
+
+
+## Built With
+
+- VSCODE
+- Github
+- Git
+
+## Getting Started
+
+
+## Authors
```

Comment:
##Changes Required!

⚠️ Please add yourself as an author on your README.md. This will help showcase your work! A good practice is to add links to:
- Your GitHub
- Your Twitter
- Your LinkedIn.

This will allow an easy way for the people that enjoy your project to find you and contact you! 🚀